src/services/rowService.js

```diff
@@ -1,13 +1,19 @@
 module.exports = (db) => {
   return {
     get(data) {
-      const query = `SELECT ${data.schemaString} FROM ${data.tableName} ${data.extendString} ${data.whereString} ${data.orderString} LIMIT ? OFFSET ?`;
+      const query = `SELECT ${data.schemaString || '*'} FROM ${
+        data.tableName
+      } ${data.extendString || ''} ${data.whereString || ''} ${
+        data.orderString || ''
+      } LIMIT ? OFFSET ?`;
+
       const statement = db.prepare(query);
       const result = statement.all(
         ...data.whereStringValues,
-        data.limit,
-        data.page,
+        data.limit || 10,
+        data.page || 0,
```

Comment:
These two ("10" and "0", e.g. default limit and offset), I think it's better to have them in constants or somewhere instead of hardcoding it in the code.